gentx/betanet-gentx/Castlenode-77ec907b9dd751a4.json

```diff
@@ -0,0 +1 @@
+{"type":"irishub/bank/StdTx","value":{"msg":[{"type":"irishub/stake/MsgCreateValidator","value":{"Description":{"moniker":"Castlenode","identity":"","website":"","details":""},"Commission":{"rate":"0.1000000000","max_rate":"0.5000000000","max_change_rate":"0.0100000000"},"delegator_address":"faa1fyxjf70lcmcu7fe3nzc4hq8glc30fq0rs335d6","validator_address":"fva1fyxjf70lcmcu7fe3nzc4hq8glc30fq0r9qmmsa","pubkey":{"type":"tendermint/PubKeyEd25519","value":"csizakYQj3TEjz2GXnG0kIdIQ93ty3WjoynJEvzYGPo="},"delegation":{"denom":"iris-atto","amount":"100000000000000000000"}}}],"fee":{"amount":null,"gas":"200000"},"signatures":[{"pub_key":{"type":"tendermint/PubKeySecp256k1","value":"AlXopivQjqEjyJMOlP6f8rZE6xLprS90AmOHfpYRAr28"},"signature":"3ULZwJMYS+TwPNLhuAuoJ3QZ1wUwBXQfZ19R1P8fGUFP3zKRLsBHhAI3DYJKpEjbMho43d5PnbJ8MMENr83hGQ==","account_number":"0","sequence":"0"}],"memo":"77ec907b9dd751a4dff0d2e25b81fa0a99b21447@185.42.117.29:26656"}}
```

Comment:
Please use/build the `mainnet` binary to generate addresses starting with `iaa` instead of `faa`.  Following instructions here: https://github.com/irisnet/betanet/tree/master/gentx